middleware/jwt.go

```diff
@@ -53,7 +53,12 @@ var JwtConfig = &jwt.GinJWTMiddleware{
 			return nil, jwt.ErrFailedAuthentication
 		}
 
-		if utils.ComparePasswords(user.Password, []byte(password)) {
+		if user.Password == nil {
+			log.Println("User password in database is null indicating they use oauth and not the password flow")
+			return nil, jwt.ErrFailedAuthentication
+		}
+		passwordMatches, _ := utils.ComparePasswords(user.Password, []byte(password))
```

Comment:
Since we've started returning the error from `ComparePasswords` now, we might as well use it here. Even though we're pretty much certain it will always be `nil` in this case, if it's returning an error I think we should use it.